uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_description/config/sensor_parameters.yaml

```diff
@@ -0,0 +1,46 @@
+depth_camera:
```

Comment:
Any specific reasons behind these positions? Also, can we change the field of view of the cameras or how is that specified?